platform-ui-skin/pom.xml

```diff
@@ -37,6 +37,7 @@
             <plugin>
                 <groupId>org.lesscss</groupId>
                 <artifactId>lesscss-maven-plugin</artifactId>
+                <version>1.7.0.1.1</version>

```

Comment:
ok :)
But I think we should do this step by step and starts by fixing the issue about navbarHeight. So I would remove the version upgrade for the lesscss maven plugin in this PR since it is not required to fix the issue. WDYT ?